src/test/java/com/bynder/sdk/query/upload/SaveMediaQueryTest.java

```diff
@@ -13,20 +17,31 @@
     public static final String EXPECTED_BRAND_ID = "brandId";
     public static final String EXPECTED_NAME = "name";
     public static final String EXPECTED_MEDIA_ID = "mediaId";
-    public static final String EXPECTED_METAPROPERTY = "metaproperty.id1=value1,metaproperty.id2=value2";
+    public static final String EXPECTED_METAPROPERTY_ID = "metapropertyId";
+    public static final String EXPECTED_OPTION_NAME = "optionName";
 
     @Test
     public void initializeSaveMediaQuery() {
         SaveMediaQuery saveMediaQuery = new SaveMediaQuery(EXPECTED_IMPORT_ID);
         saveMediaQuery.setBrandId(EXPECTED_BRAND_ID);
         saveMediaQuery.setName(EXPECTED_NAME);
         saveMediaQuery.setMediaId(EXPECTED_MEDIA_ID);
-        saveMediaQuery.setMetaproperty(EXPECTED_METAPROPERTY);
+        saveMediaQuery.setMetaproperty(EXPECTED_METAPROPERTY_ID, EXPECTED_OPTION_NAME);
 
         assertEquals(EXPECTED_IMPORT_ID, saveMediaQuery.getImportId());
         assertEquals(EXPECTED_BRAND_ID, saveMediaQuery.getBrandId());
         assertEquals(EXPECTED_NAME, saveMediaQuery.getName());
         assertEquals(EXPECTED_MEDIA_ID, saveMediaQuery.getMediaId());
-        assertEquals(EXPECTED_METAPROPERTY, saveMediaQuery.getMetaproperty());
+        assertEquals(1, saveMediaQuery.getMetaproperties().size());
```

Comment:
We could still test the actual contents by doing something like:
```
private static final Map<String, String> EXPECTED_METAPROPERTIES = new HashMap<>();
static {
    EXPECTED_METAPROPERTIES.put("metapropertyId", "optionName");
    // or make it a bit bigger than just 1 element
}
```
```
saveMediaQuery.setMetaproperties(EXPECTED_METAPROPERTIES);
assertTrue(EXPECTED_METAPROPERTIES.equals(saveMediaQuery.getMetaproperties()));`
```

And the same could be applied to `UploadQueryTest`